fighthealthinsurance/pubmed_tools.py

```
@@ -127,39 +128,37 @@ async def find_pubmed_articles_for_denial(
+        # Build the set of queries: base query + any microsite search terms
+        queries: Set[str] = {query}
+        if denial.microsite_slug:
+            try:
+                microsite = get_microsite(denial.microsite_slug)
+                if microsite and microsite.pubmed_search_terms:
+                    if len(microsite.pubmed_search_terms) > 0:
+                        logger.debug(
+                            f"Adding {len(microsite.pubmed_search_terms)} microsite search terms for {denial.microsite_slug}"
+                        )
+                        queries.update(microsite.pubmed_search_terms)
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)